### PR TITLE
Added assingment

### PR DESCRIPTION
**Pull request by ruchitamamulkar**

### CODE REVIEW

File: DataEngineering/sc.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import json, requests
+import sqlite3
+
+
+conn = sqlite3.connect('Bike.db')
+c = conn.cursor()
+c.execute('CREATE TABLE if not exists realtimewithT (Id,value INTEGER, timestamp)')
```
Comment (dwividha) [inline DataEngineering/sc.py:7]: How would the user make a query like 'what was the availability of bikes at bikestand X at 5PM yesterday?'
Try to capture as much data as possible from the API request here - all data is valuable.

File: DataEngineering/sc.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+url = "https://api.tfl.gov.uk/bikepoint"
+response = requests.get(url)
+data = response.text
+parsed = json.loads(data)
+#print (parsed)
+
+rn= (len(parsed))
+for i in range(rn):
```
Comment (dwividha) [inline DataEngineering/sc.py:19]: Where is i being used? You are only storing data from the 0th element here - what about the rest of the bikestands?

File: DataEngineering/sc.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    lon=(parsed[0]['lon'])
+    
+    abc=(parsed[0]['additionalProperties'])[0]
+    value=(abc['value'])
+    timestamp=(abc['modified'])
+    c.execute('INSERT INTO realtime (Id,location,lat,lon) VALUES(?,?,?,?)',(Id,location,lat,lon))
+    conn.commit()
+    
+    c.execute('INSERT INTO realtimeWithT (Id,value,timestamp) VALUES(?,?,?)',(Id,value,timestamp))
+    conn.commit()
```
Comment (dwividha) [inline DataEngineering/sc.py:32]: This code (when corrected) can be used to only create and insert to the table. Please refer to point 4 under 'Assessment Criteria' under 'ownership'.

File: DataEngineering/README.md [hunk 4]
```diff
@@ -1,43 +1,14 @@
...
+4.sqlite3 and all required library installed
+
+How to run 
+1.Open terminal and type python3 sc.py and press enter to run the code 
+2.For cron tab
+  1.Open terminal(ubantu) type crontab -e and edit that file write code which is written in crontab.txt file and press ctrl+o,enter,ctrl+x
+Note:In cron file after python3 put the path of the sc.py file 
+  2.Check the output in the bike.db
+3.How to open bike db
+  1.Drag and drop the file/files of Bike.db in the sqlite and check for tables
```
Comment (dwividha) [inline DataEngineering/README.md:14]: Markdown (md) files have a specific syntax that make them readable - refer to [this](https://www.markdownguide.org/basic-syntax/)

File: DataEngineering/README.md [hunk 5]
```diff
@@ -1,43 +1,14 @@
...
-1. Clone this repository
-2. Remove the assignment README’s (and all folders) and start developing your project in it
-3. Add a readme that details on exactly how to run the code
-4. Raise a Pull Request
\ No newline at end of file
+Requirements
+1.Internet connection to load data from API
+2.Ubuntu(for running Cron jobs)
+3.Python3 
+4.sqlite3 and all required library installed
```
Comment (dwividha) [inline DataEngineering/README.md:5]: As a complete package, a good practice is to provide a requirements.txt file which contains all the libraries required to run this package within the folder as well.

**Comment by dwividha** [comment]
Thanks for trying out our Assignment @ruchitamamulkar!
Closing this PR.

**Comment by ruchitamamulkar** [comment]
Hi,
I will look into this questions of Saturday and Sunday and update you on
the same
Sorry for late reply
Thank you

**Pull request closed without merging**